apps/ui/src/components/AppLink.vue

```diff
@@ -14,19 +10,7 @@ function normalize(to: any) {
 </script>
 
 <template>
-  <router-link
-    v-slot="{ isActive, href, navigate }"
-    v-bind="props"
-    :to="normalize(props.to)"
-    custom
-  >
-    <a
-      v-bind="$attrs"
-      :href="href"
-      :class="{ [`${activeClass}`]: isActive }"
-      @click="navigate"
-    >
-      <slot />
-    </a>
+  <router-link :to="normalize(props.to)">
```

Comment:
It won't be possible to pass other props to router-link?

Reply:
`router-link` is the only child in this component so `$attrs` will be bound to it automatically.